package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nemo",
-  "version": "4.2.2",
+  "version": "4.2.3",
```

Comment:
Please leave the version property unchanged.

Reply:
I'm going to make the change. FYI, current Nemo version is `4.2.2` but the last changelog entry is `4.2.1`.